app/assets/stylesheets/landing-responsive.sass

```diff
@@ -1,4 +1,4 @@
-@include media-breakpoint-down(md)
+// @include media-breakpoint-down(md)
```

Comment:
I comment this line for a moment because crash the app, causing this error: `Error: no mixin named media-breakpoint-down` However, no change in the normal layout because it is not in use, in the application.scss does not import the file landing.scss